src/CollectionFactory.php

```diff
@@ -34,6 +34,15 @@ private function build(array $extra = [], string $creationType = 'create'): Coll
     {
         return collect()
             ->times($this->times)
-            ->transform(fn($value, $key) => $this->modelClass::$creationType(array_merge($this->modelsDefaultData[$key], $extra)));
+            ->transform(fn($value, $key) => $this->modelClass::$creationType(
```

Comment:
I believe when we have a `prepareModelDataForBuild` method in the `BaseFactory` class, we can use it when the `times` method there is called. This way we should already have the correct defaultData in the `CollectionFactory`, right?